opentech/apply/funds/tables.py

```diff
@@ -150,3 +152,102 @@ def __init__(self, *args, exclude=list(), **kwargs):
 
 class SubmissionFilterAndSearch(SubmissionFilter):
     query = filters.CharFilter(field_name='search_data', lookup_expr="icontains", widget=forms.HiddenInput)
+
+
+class RoundsTable(tables.Table):
+    title = tables.LinkColumn('funds:rounds:detail', args=[A('pk')], orderable=True, text=lambda record: record.title)
+    fund = tables.Column(accessor=A('specific.fund'))
+    lead = tables.Column()
+    start_date = tables.Column()
+    end_date = tables.Column()
+    progress = tables.Column()
+
+    class Meta:
+        fields = ('title', 'fund', 'lead', 'start_date', 'end_date', 'progress')
+
+    def render_lead(self, value):
+        return format_html('<span>{}</span>', value)
+
+    def render_progress(self, record):
+        return f'{record.progress}%'
+
+    def _field_order(self, field, desc):
+        return getattr(F(f'{field}'), 'desc' if desc else 'asc')(nulls_last=True)
+
+    def order_start_date(self, qs, desc):
+        return qs.order_by(self._field_order('start_date', desc)), True
+
+    def order_end_date(self, qs, desc):
+        return qs.order_by(self._field_order('end_date', desc)), True
+
+    def order_fund(self, qs, desc):
+        funds = ApplicationBase.objects.filter(path=OuterRef('parent_path'))
+        qs = qs.annotate(
+            parent_path=Left(F('path'), Length('path') - ApplicationBase.steplen, output_field=CharField()),
+            fund=Subquery(funds.values('title')[:1]),
+        )
+        return qs.order_by(self._field_order('fund', desc)), True
+
+    def order_progress(self, qs, desc):
+        return qs.order_by(self._field_order('progress', desc)), True
+
+
+# TODO remove in django 2.1 where this is fixed
+F.relabeled_clone = lambda self, relabels: self
+
+
+# TODO remove in django 2.1 where this is added
+class Left(Func):
```

Comment:
@frjo These are the fixes i was hoping to have from 2.1. Not much to include here but worth remembering for future reference.

Left Function
https://docs.djangoproject.com/en/2.1/ref/models/database-functions/#left

Fixes an inssue with `F` and `OuterRef`
https://code.djangoproject.com/ticket/29142